src/main/java/uk/gov/justice/digital/delius/jpa/standard/repository/OASYSAssessmentRepository.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.digital.delius.jpa.standard.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+import uk.gov.justice.digital.delius.jpa.standard.entity.OASYSAssessment;
+
+import java.util.Optional;
+
+@Repository
+public interface OASYSAssessmentRepository extends JpaRepository<OASYSAssessment, Long> {
+
+    @Query("select assessment from OASYSAssessment assessment " +
+        "where assessment.softDeleted = 0 " +
+        "and assessment.offenderId = :offenderId " +
+        "and assessment.lastUpdatedDate = \n" +
```

Comment:
There are a few dates we could use here according to the schema, we should check with the Delius team which is the correct one, if you haven't.

Reply:
https://github.com/ministryofjustice/delius/blob/b57df1981aaee802dc951229f41e08c8aa11b52f/NDelius-ejb/src/main/java/uk/co/bconline/ndelius/dao/assessments/AssessmentsDAOBean.java#L283
uses assessmentDate followed by editedDate. Good spot

Reply:
I've managed this using a JPA method name. The same is true of the OGRS assessments so that will be next

https://github.com/ministryofjustice/delius/blob/b57df1981aaee802dc951229f41e08c8aa11b52f/NDelius-ejb/src/main/java/uk/co/bconline/ndelius/dao/sentencing/EventDAOBean.java#L2920